community/bolt/src/docs/dev/serialization.asciidoc

```diff
@@ -270,6 +270,8 @@ D4 14 01 02  03 04 05 06  07 08 09 00  01 02 03 04
 === Maps
 Maps are sized sequences of pairs of values and permit a mixture of types within the same map.
 The size of a map denotes the number of pairs within that map, not the total packed byte size.
+Keys are unique within a map, however the protocol notably technically allows duplicate keys to be sent.
+If duplicate keys are sent, this is a violation of the protocol and an error will occur.
```

Comment:
I read this two lines and feel ambiguous. I feel the reason is the first "protocol" and the "second" protocol actually means different, otherwise how the protocol both allows duplicate keys and also disallows duplicate keys?